src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,81 @@
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">
-                illo est ratione doloremque quia maiores aut
-              </p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </>
+  return (
+    <table className="table is-narrow is-fullwidth">
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>
+            <span className="icon">
+              <i className="fas fa-check" />
+            </span>
+          </th>
+          <th>Title</th>
+          <th> </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {filteredTodos.map(todo => {
+          const isSelected = activeTodo?.id === todo.id;
+
+          return (
+            <tr
+              key={todo.id}
+              data-cy="todo"
+              className={cn({
+                'has-background-info-light': isSelected,
+              })}
+            >
+              <td className="is-vcentered">{todo.id}</td>
+              <td className="is-vcentered">
+                {todo.completed && (
+                  <span className="icon" data-cy="iconCompleted">
+                    <i className="fas fa-check" />
+                  </span>
+                )}
+              </td>
+              <td className="is-vcentered is-expanded">
+                <p
+                  className={
+                    todo.completed ? 'has-text-success' : 'has-text-danger'
+                  }
+                >
+                  {todo.title}
+                </p>
+              </td>
+              <td className="has-text-right is-vcentered">
+                <button
+                  data-cy="selectButton"
+                  className="button"
+                  type="button"
+                  onClick={() => dispatch(setCurrentTodo(todo))}
+                >
+                  <span className="icon">
+                    <i
+                      className={
+                        !activeTodo ? 'far fa-eye' : 'far fa-eye-slash'
```

Comment:
The conditional class assignment for the eye icon seems to be incorrect. The logic currently assigns 'far fa-eye' when `!activeTodo` is true and 'far fa-eye-slash' otherwise. This logic might be reversed if the intention is to show an eye icon when there is no active todo and an eye-slash icon when there is an active todo. Please verify and adjust the logic if necessary.